t/base/pat.t

```diff
@@ -1,9 +1,16 @@
 #!./perl
 
+# t/base/pat.t - check base regex for $_
+
 print "1..2\n";
 
-# first test to see if we can run the tests.
+# First test to see if we can run the tests.
 
 $_ = 'test';
-if (/^test/) { print "ok 1 - match regex\n"; } else { print "not ok 1 - match regex\n";}
-if (/^foo/) { print "not ok 2 - match regex\n"; } else { print "ok 2 - match regex\n";}
+/^test/
+  ? ( print "ok 1 - match regex\n" )
```

Comment:
the `?:` operators should be reserved for strict functional expressions -- that is, they should not be used solely to achieve side effects.

Reply:
I see. I will return the form.

Reply:
I found the same using `?:` for tests t/base/num.t however.